src/bin/sccache-dist/build.rs

```diff
@@ -410,11 +410,9 @@ impl OverlayBuilder {
                         output: compile_output,
                         outputs,
                     })
-
                     // Bizarrely there's no way to actually get any information from a thread::Result::Err
                 })
                 .join()
-                .unwrap_or_else(|_e| Err(anyhow!("Build thread exited unsuccessfully")))
```

Comment:
Is there a particular reason to remove the error mapping? `join` in of `scoped` also returns a `Result`

Reply:
The std version's scoped thread does not return Result, so we should unwrap only once. However I noticed that it may be better to move error mapping here from outside of the scoped thread. I will fix it.

Reply:
Fixed.

Reply:
Thank you!